README.md

```diff
@@ -41,10 +41,17 @@ Basically, you could manage all data streams with a `Listing` component, which i
 It's really flexible and useful to display the paged list entities and reflect the network status changes in the UI.
 
 Fountain provides two ways to generate a `Listing` component from paged services:    
-1. **Network support:** Provides a `Listing` based on a service that uses Retrofit and RxJava. Note entities won't be saved in memory nor disk.
-1. **Cache + Network support:** Provides a `Listing` with cache support using a service based on Retrofit and RxJava, and a [`DataSource`](https://developer.android.com/reference/android/arch/paging/DataSource) for caching the data.
+1. [**Network support:**](#network-support) Provides a `Listing` based on a common Retrofit service implementation.
+Note entities won't be saved in memory nor disk.
+1. [**Cache + Network support:**](#cache--network-support) Provides a `Listing` with cache support using a common Retrofit service implementation, and a [`DataSource`](https://developer.android.com/reference/android/arch/paging/DataSource) for caching the data.
 We recommend you use [Room](https://developer.android.com/topic/libraries/architecture/room) to provide the `DataSource`, because it will be easier. However, you could use any other `DataSource`.
 
+Fountain supports 2 types of Retrofit service adapters:
+- A [RxJava2 retrofit adapter.](https://github.com/square/retrofit/tree/master/retrofit-adapters/rxjava2)
+- A [Coroutine retrofit adapter.](https://github.com/JakeWharton/retrofit2-kotlin-coroutines-adapter)
+
+It also supports not using an adapter, you can also work with a simple Retrofit [call](https://square.github.io/retrofit/2.x/retrofit/retrofit2/Call.html).
```

Comment:
Maybe `It also supports not using any of them, as you could work with a simple...`??